csv-to-db/docker-compose.yml

```diff
@@ -9,3 +9,6 @@ services:
       - POSTGRES_PASSWORD=kaogeek
     ports:
       - "5432:5432"
+    volumes:
+      - ./create_report_table.sql:/docker-entrypoint-initdb.d/init.sql
+      - ../output_example_v2.csv:/usr/src/output_example_v2.csv
```

Comment:
I don't think we need this as the thing we import is from [here](https://raw.githubusercontent.com/napatswift/thailand-budget-pdf2csv/main/csv-extraction/napatswift-coordintes/output/65_thailand_budget_extracted_b4_cleansing_28Jul21.csv).